lk2nd/device/dts/msm8952/msm8937-nokia-ple.dts

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	qcom,msm-id = <QCOM_ID_MSM8937 0x2000>,
+		      <QCOM_ID_MSM8937 0x3000>;
+	qcom,board-id = <0x9B 0>;
+};
+
+&lk2nd {
+	ple {
+		model = "HMD Global Nokia 6 (ple)";
+		compatible = "nokia,ple";
+		qcom,msm-id = <QCOM_ID_MSM8937 0x2000>,
+			      <QCOM_ID_MSM8937 0x3000>;
+		lk2nd,match-device = "PLE";
+
+		lk2nd,dtb-files = "msm8937-nokia-ple";
+
+		gpio-keys {
+			compatible = "gpio-keys";
+			up {
+				lk2nd,code = <KEY_VOLUMEUP>;
+				gpios = <&tlmm 91 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;
+			};
+		};
+
+		panel {
+			compatible = "nokia,ple-panel", "lk2nd,panel";
+
+			fih,mdss_dsi_hx8394a_720p_video {
+				compatible = "nokia,ple-hx8394a";
+			};
+			fih,mdss_dsi_ili7807e_1080p_video {
```

Comment:
is this device 720p or 1080p?

Reply:
<i>* Looks through the device tree *</i>
Oh, oops, my mistake.